cleverhans/attacks.py

```diff
@@ -505,6 +521,20 @@ def parse_params(self,
         return True
 
 
+class BasicIterativeMethod(ProjectedGradientDescent):
```

Comment:
do we actually need different subclasses? The only difference is the value of one parameter, right?

Reply:
talked with Nicolas offline. It makes sense to keep the different subclasses so that there is one class per published paper and for backwards compatibility.